src/devices/Amg88xx/Register.cs

```diff
@@ -0,0 +1,953 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Amg88xx
+{
+    /// <summary>
+    /// Defines the addresses of the AMG88xx registers
+    /// </summary>
+    public enum Register : byte
```

Comment:
Any reason why you make the Register public? Should it be internal? I don't really see any method using it.

Reply:
This seems to be not need to be exposed in the public surface. Internal should be enough